src/main/java/blackjack/domain/OwnCards.java

```diff
@@ -0,0 +1,43 @@
+package blackjack.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OwnCards {
+
+    private final int BLACKJACK = 21;
+    private final int ACE_NUMBER_TEN = 10;
+    private final int ACE_NUMBER_ONE = 1;
+
+    private final List<Card> ownCards;
+
+    public OwnCards() {
+        this.ownCards = new ArrayList<>();
+    }
+
+    public int getTotalScore() {
+        int total = ownCards.stream().mapToInt(Card::getCardNumber).sum();
+
+        if (total <= BLACKJACK - ACE_NUMBER_TEN && ownCards.stream().anyMatch(Card::isAceCard)) {
+            return total + ACE_NUMBER_TEN - ACE_NUMBER_ONE;
+        }
```

Comment:
아래와 같이 변경해보면 어떨까요? 🤗
```suggestion
        boolean hasAceCard = ownCards.stream().anyMatch(Card::isAceCard);
        if (total + ACE_NUMBER_TEN <= BLACKJACK && hasAceCard) {
            return total + ACE_NUMBER_TEN - ACE_NUMBER_ONE;
        }
```